### PR TITLE
Write as GeoTIFF + Sidecar YAML (DEADM 1070)

### PR DESCRIPTION
**Pull request by dunkgray**
This is for the requirement write Output Fractional Cover Datasets as GeoTIFF + Sidecar YAML.

[View DEADM-1070 in Jira](https://gajira.atlassian.net/secure/RapidBoard.jspa?rapidView=423&view=planning&selectedIssue=DEADM-1070)

### CODE REVIEW

File: fc/fc_app.py [hunk 1]
```diff
@@ -565,6 +647,79 @@ def run(index,
...
+    :param bands: a list of bands/measurements
+    :param sep: the separator between the base name and the band.
+    :return: filenames
+    """
+    base, ext = os.path.splitext(filename)
+    assert ext == '.tif'
+    filenames = {}
+    for band in bands:
+        build = Path(base + sep + band + ext)
+        filenames[band] = build.absolute().as_uri()
```
Comment (andrewdhicks) [inline fc/fc_app.py]: If this was relative to the yaml instead, we could avoid manipulating the file before S3 upload

Comment (dunkgray) [reply fc/fc_app.py]: Good idea Andrew. I'll look into doing this.

File: fc/fc_app.py [hunk 2]
```diff
@@ -159,10 +169,12 @@ def _make_fc_tasks(index: Index,
...
     tiles_in = workflow.list_tiles(product=input_product.name, **query)
-    _LOG.info(f"{len(tiles_in)} {input_product.name} tiles in {repr(query)}")
+    #_LOG.info(f"{len(tiles_in)} {input_product.name} tiles in {repr(query)}")
+    _LOG.info('%d %s tiles in %s', len(tiles_in), input_product.name, str(repr(query)))
+    # _LOG.info('Found %d tasks', num_tasks_saved)
     tiles_out = workflow.list_tiles(product=output_product.name, **query)
-    _LOG.info(f"{len(tiles_out)} {output_product.name} tiles in {repr(query)}")
-
+    #_LOG.info(f"{len(tiles_out)} {output_product.name} tiles in {repr(query)}")
+    _LOG.info('%d %s tiles in %s', len(tiles_out), output_product.name, str(repr(query)))
```
Comment (omad) [inline fc/fc_app.py]: Why the change of logging method?

It used to be necessary to split logging string creation like this, but `f-strings` are now considered just as fast and completely acceptable.

Comment (dunkgray) [reply fc/fc_app.py]: Travis was failing here. This was a fix to pass Travis at the time. I wasn't keen on the change either. The root issue with Travis is fixed now.
